lib/rspec/mocks/minitest_integration.rb

```diff
@@ -0,0 +1,15 @@
+require 'rspec/mocks'
+
+Minitest::Expectation.class_eval do
+  def to(*args, &block)
+    ::RSpec::Mocks::ExpectationTarget.new(target).to(*args, &block)
+  end
+
+  def not_to(*args, &block)
+    ::RSpec::Mocks::ExpectationTarget.new(target).not_to(*args, &block)
+  end
+
+  def to_not(*args, &block)
+    ::RSpec::Mocks::ExpectationTarget.new(target).to_not(*args, &block)
+  end
+end
```

Comment:
If we're going to provide minitest integration, I think we should go all the way and provide a file that handles _all_ of the integration instead of just this bit.  Our [cuke](https://github.com/rspec/rspec-mocks/blob/master/features/outside_rspec/minitest.feature) shows an example of the sort of code that a user has to write to integrate the two libraries, and that setup/verify/teardown logic should go in this file as well.  Then minitest users can just do `require 'rspec/mocks/minitest_integration'` and it'll handle all of the integration :).